### PR TITLE
Api search

### PR DESCRIPTION
**Pull request by Madhuri97**
done with the search page

### CODE REVIEW

File: project1/templates/login.html [hunk 1]
```diff
@@ -48,6 +48,44 @@
...
+            "<th class='bg-secondary text-white'> {{ 'View Book Page' }}</th>";
+            for(var i = 0; i < data.length; i++) {
+              htmltext = htmltext + "<tr>" + "<td>"+data[i]["isbn"] +"</td>"+ "<td>" + data[i]["title"] +"</td>"+
+              "<td>" + data[i]["author"] +"</td>"+ "<td>" + data[i]["year"] +"</td><td><button onclick = 'bookdisplay(this)' id = "+
+              data[i]["isbn"] + ">View Book</button></td></tr>"
+            }
+      htmltext = htmltext + " </tbody> </table> </div></div>"
+      document.getElementById("htmltext").innerHTML = htmltext;
+    } else if(xhr.status == 404) {
+      console.log("Error: Page Not Found");
```
Comment (ChaitanyaPuritipati) [inline project1/templates/login.html:81]: This error should be visible to the user right ?? console.log() won't solve the purpose. It just only displays in the console.

File: project1/templates/login.html [hunk 2]
```diff
@@ -48,6 +48,44 @@
...
+    } else {
+      console.log("Error: Please Try again");
```
Comment (ChaitanyaPuritipati) [inline project1/templates/login.html:83]: Same as the above. Please display the error to the user also

**Review by ChaitanyaPuritipati — CHANGES_REQUESTED** [review]

**Review by ChaitanyaPuritipati — CHANGES_REQUESTED** [review]
I didn't see search API in the application.py. Please push that also onto this branch

**Comment by Madhuri97** [comment]
> I didn't see search API in the application.py. Please push that also onto this branch

https://github.com/Madhuri97/project-1-105/pull/10/commits/b9d43e46fc6d4819b02c8f06ad3e9b2900fe03ed- I had pushed previously as you commented I pushed that file again to the repo

File: project1/templates/login.html (hunk 1 above)
Comment (Madhuri97) [reply project1/templates/login.html:81]: https://github.com/Madhuri97/project-1-105/pull/10/commits/bd4a9c393d9defddc16c17495fb863034e168d1f

File: project1/templates/login.html (hunk 2 above)
Comment (Madhuri97) [reply project1/templates/login.html:83]: https://github.com/Madhuri97/project-1-105/pull/10/commits/bd4a9c393d9defddc16c17495fb863034e168d1f

**Pull request merged**